spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -3,20 +3,17 @@
 
 [partintro]
 --
-This section goes into more detail about how you can work with Spring Cloud Stream. It covers topics
-such as creating and running stream applications.
+This section goes into more detail about how you can work with Spring Cloud Stream.
+It covers topics such as creating and running stream applications.
 --
 
 == Introducing Spring Cloud Stream
+Spring Cloud Stream is a framework for building message-driven microservice applications.
+Spring Cloud Stream builds upon Spring Boot and uses Spring Integration to provide connectivity to message brokers.
```

Comment:
I just think we should have a simple rationale for using Boot. Instead of DevOps we could say: 
'Spring Cloud Stream builds upon Spring Boot to create standalone, production-grade Spring Applications, and uses Spring Integration etc.'